docs/src/tutorials/pluscal-tutorial.md

```diff
@@ -0,0 +1,241 @@
+# Tutorial on checking Pluscal specifications with Apalache
+
+**Difficulty: Blue trail – Easy**
+
+In this short tutorial, we show how to annotate a [Pluscal specification][] of
+the Bakery algorithm, to check it with Apalache. Further, we check mutual
+exclusion by bounded model checking for bounded executions. Moreover, we
+automatically prove mutual exclusion by induction and bounded model checking
+for unbounded executions.
+
+We only focus on the part
+related to Apalache. If you want to understand the Bakery algorithm and its
+specification, check the comments in the original [Pluscal specification][].
+
+## Related documents
+
```

Comment:
Are these meant to be prerequisites to this tutorial, or further reading that
can be pursued *after* completing this tutorial?

If the former, I'd suggest making it clear that the reader should already be
comfortable with the relevant material. If the latter, I suggest moving them to
the end under "Further reading". Either way, I suspect it will help readers to
indicate what the nature of the relation is :)